dspace-api/src/main/java/org/dspace/browse/ItemCounter.java

```diff
@@ -63,21 +75,40 @@ public ItemCounter(Context context) throws ItemCountException {
         this.dao = ItemCountDAOFactory.getInstance(this.context);
         this.itemService = ContentServiceFactory.getInstance().getItemService();
         this.configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+        this.showStrengths = configurationService.getBooleanProperty("webui.strengths.show", true);
+        this.useCache = configurationService.getBooleanProperty("webui.strengths.cache", true);
     }
 
     /**
-     * Get the count of the items in the given container.  If the configuration
-     * value webui.strengths.cache is equal to 'true' this will return the
-     * cached value if it exists.  If it is equal to 'false' it will count
-     * the number of items in the container in real time.
+     * Get the singular instance of a class.
+     * It creates a new instance at the first usage of this method.
+     *
+     * @return instance af a class
+     * @throws ItemCountException when error occurs
+     */
+    public static ItemCounter getInstance() throws ItemCountException {
+        if (instance == null) {
+            instance = new ItemCounter(ContextUtil.obtainCurrentRequestContext());
+        }
+        return instance;
+    }
+
+    /**
+     * Get the count of the items in the given container. If the configuration
+     * value webui.strengths.show is equal to 'true' this method will return all
+     * archived items. If the configuration value webui.strengths.cache
+     * is equal to 'true' this will return the cached value if it exists.
+     * If it is equal to 'false' it will count the number of items
+     * in the container in real time.
      *
      * @param dso DSpaceObject
      * @return count
      * @throws ItemCountException when error occurs
      */
     public int getCount(DSpaceObject dso) throws ItemCountException {
-        boolean useCache = configurationService.getBooleanProperty(
-            "webui.strengths.cache", true);
+        if (!showStrengths) {
+            return 0;
```

Comment:
Rather than returning zero (0), I'd recommend this return `-1` if it is disabled.  This way you can actually tell if it's disabled or not.  If it returns zero, it could be that the Collection/Community is empty.

(I notice that the current behavior is that if the value is zero, it just doesn't display.  I don't think it behaved that way in 6.x.  In 6.x, if the collection was empty it showed a count of `0`.)